ckanext/validation/logic.py

```diff
@@ -656,26 +656,31 @@ def _run_sync_validation(resource_id, local_upload=False, new_resource=True):
         {u'ignore_auth': True},
         {u'resource_id': resource_id})
 
-    report = validation['report']
+    if validation['report']:
```

Comment:
Is there an instance when `validation["report"]` is not set? This is our own ckanext-validation object so even if frictionless doesn't return an object for some reason we could store a custom one like `{"valid": False, "errors": [...]}` or something similar. IIUC in the goodtables-based code we always had a `report` key